evalml/automl/automl_search.py

```diff
@@ -607,10 +607,18 @@ def _compute_cv_scores(self, pipeline, X, y):
         start = time.time()
         cv_data = []
         logger.info("\tStarting cross validation")
+        warnings.filterwarnings("ignore", lineno=665)
         for i, (train, test) in enumerate(self.data_split.split(X, y)):
             logger.debug(f"\t\tTraining and scoring on fold {i}")
             X_train, X_test = X.iloc[train], X.iloc[test]
             y_train, y_test = y.iloc[train], y.iloc[test]
+            if self.problem_type in [ProblemTypes.BINARY, ProblemTypes.MULTICLASS]:
+                diff_train = set(y).difference(set(y_train))
```

Comment:
I think we should use `np.setdiff1d` - it will be faster than `set.difference` since these are already series. Might need to change the `if diff_train` to get this to work though. 

![image](https://user-images.githubusercontent.com/41651716/94475049-88686100-019c-11eb-9d8f-b10cda830e71.png)